solutions/instance/main.tf

```diff
@@ -58,9 +58,13 @@ locals {
   existing_secrets_manager_instance_crn_split = var.existing_secrets_manager_instance_crn != null ? split(":", var.existing_secrets_manager_instance_crn) : null
   existing_secrets_manager_instance_guid      = var.existing_secrets_manager_instance_crn != null ? element(local.existing_secrets_manager_instance_crn_split, length(local.existing_secrets_manager_instance_crn_split) - 3) : null
   existing_secrets_manager_instance_region    = var.existing_secrets_manager_instance_crn != null ? element(local.existing_secrets_manager_instance_crn_split, length(local.existing_secrets_manager_instance_crn_split) - 5) : null
+
+  # tflint-ignore: terraform_unused_declarations
+  validate_sm_crn = length(local.service_credential_secrets) > 0 && var.existing_secrets_manager_instance_crn == null ? tobool("`var.existing_secrets_manager_instance_crn` is required when adding service credentials to a secrets manager secret.") : false
```

Comment:
`var.existing_secrets_manager_instance_crn` means nothing in projects land. 

Change it to:

> `existing_secrets_manager_instance_crn` input is required when adding service credentials to a secrets manager secret

Reply:
addressed